src/kernel-install/90-loaderentry.install.in

```diff
@@ -115,6 +115,43 @@ install -m 0644 "$KERNEL_IMAGE" "$KERNEL_DEST" || {
 }
 chown root:root "$KERNEL_DEST" || :
 
+if [ -n "$KERNEL_INSTALL_CONF_ROOT" ]; then
+    if [ -f "$KERNEL_INSTALL_CONF_ROOT/devicetree" ]; then
+        read -r DEVICETREE <"$KERNEL_INSTALL_CONF_ROOT/devicetree"
+    fi
+elif [ -f /etc/kernel/devicetree ]; then
+    read -r DEVICETREE </etc/kernel/devicetree
+elif [ -f /usr/lib/kernel/devicetree ]; then
+    read -r DEVICETREE </usr/lib/kernel/devicetree
+fi
+if [ -n "$DEVICETREE" ]; then
+    for prefix in \
+        "/boot/dtb-$KERNEL_VERSION" \
+        "/boot/dtbs/$KERNEL_VERSION" \
+        "/lib/firmware/$KERNEL_VERSION/device-tree" \
+        "/lib/linux-image-$KERNEL_VERSION" \
+        "/lib/modules/$KERNEL_VERSION/dtb"
```

Comment:
The kernel also uses a fixed `/lib`-prefixed path for modules, so I guess it's OK to just do that here too.